packages/taquito-rpc/src/rpc-client-modules/rpc-cache.ts

```diff
@@ -1224,4 +1227,25 @@ export class RpcClientCache implements RpcClientInterface {
       return response;
     }
   }
+
+  /**
+   * @description List the prevalidated operations in mempool (accessbililty of mempool depends on each rpc endpoint)
+   * @param args has 5 optional properties. We support version 1 with new encoding as version 0 will be deprecated soon. The rest of the properties is to filter pending operatoins response
+   * @default args { version: '1', applied: true, refused: true, outdated, true, branchRefused: true, branchDelayed: true, validationPass: undefined }
```

Comment:
typo `pending operatoins response`

Reply:
this one is still here

Reply:
Are we seeing differently?
<img width="1531" alt="image" src="https://user-images.githubusercontent.com/106410553/228641841-ba2630fa-25a8-4854-8ef1-dcf86c4ad9c7.png">

Reply:
I dont see it as outdated and if i go to the file in the PR its still there :O 
i have had this a couple times too recently...

Reply:
coupld be because of the rebase problem

Reply:
nevermind 👀 i just cant follow the link attached to this comment